DOL.WHD.Section14c.Web/src/modules/components/mainTopNavControl/mainTopNav.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Directive, ElementRef, Injector } from '@angular/core';
+import { UpgradeComponent } from '@angular/upgrade/static';
+
+export const mainTopNav = {
+  restrict: 'EA',
+  template: require('./mainTopNavControlTemplate.html'),
+  controller: 'mainTopNavControlController',
+  scope: { admin: '=' },
+  controllerAs: 'vm'
+};
+
+@Directive({
+  selector: 'main-top-nav'
+})
+export class mainTopNavDirective extends UpgradeComponent {
+  constructor(elementRef: ElementRef, injector: Injector) {
+    super('mainTopNavControl', elementRef, injector);
+  }
+}
```

Comment:
This creates a new Angular directive that is an upgrade of the AngularJS `mainTopNavControl` component.  The new Angular directive is called `main-top-nav`.

Reply:
👍